src/hotspot/os/linux/os_linux.hpp

```diff
@@ -426,6 +392,39 @@ class os::Linux {
   }
 
   static void* resolve_function_descriptor(void* p);
+
+#ifdef __GLIBC__
+  // os::Linux::get_mallinfo() hides the complexity of dealing with mallinfo() or
+  // mallinfo2() from the user. Use this function instead of raw mallinfo/mallinfo2()
+  // to keep the JVM runtime-compatible with different glibc versions.
+  //
+  // mallinfo2() was added with glibc (>2.32). Legacy mallinfo() was deprecated with
+  // 2.33 and may vanish in future glibcs. So we may have both or either one of
+  // them.
+  //
+  // mallinfo2() is functionally equivalent to legacy mallinfo but returns sizes as
+  // 64-bit on 64-bit platforms. Legacy mallinfo uses 32-bit fields. However, legacy
+  // mallinfo is still perfectly fine to use if we know the sizes cannot have wrapped.
+  // For example, if the process virtual size does not exceed 4G, we cannot hav
```

Comment:
Minor nit: the e is wrapped onto the next line.

Reply:
Huch. Thanks!